src/styles/page-template/block-areas/index.njk

```diff
@@ -19,7 +19,7 @@ stylesheets:
     <link href="{{ getFingerprint('stylesheets/main-ie8.css') }}" rel="stylesheet" media="all">
     <script src="{{ getFingerprint('javascripts/head-ie8.js') }}"></script>
   <![endif]-->
-  <script src="/javascripts/vendor/modernizr.js"></script>
+  <script src="{{ getFingerprint('javascripts/vendor/modernizr.js') }}"></script>
```

Comment:
Since we're hashing everything in the vendor folder, we probably need to use `getFingerprint` for `iframeResizer`, and adjust the import in `src/javascripts/head-ie8.js` (and probably some other bits I've missed).

Or, do we want to limit the hashing to `modernizr`?

Reply:
Good shout, let me limit it to `modernizr`

Reply:
@domoscargin Done 👍